airflow/bin/cli.py

```diff
@@ -152,16 +233,73 @@ def variables(args):
             print(var)
         except ValueError as e:
             print(e)
+    if args.delete:
```

Comment:
You can now add/remove variables from the cli. We could use it to automate some of the configuration that we do through the UI today.